### ISSUE TITLE
change getCharacterizationResults to fetch by generationID

### ISSUE THREAD

**Issue by jhardin29** [OHDSI/ROhdsiWebApi#267]
Currently getCharacterizationResults exports using the characterization id, however if a single result has multiple runs resulting in many records, the pulling back generates many extra unneeded records. 

It would be useful if you could just pass in a generation id of the desired results.

Tagging @chrisknoll because we chatted about this feature enhancement

**Comment by chrisknoll** [comment]
Right: it's not clear if the getCharacterizationResults is actually fetching the results for all generations at one time (which could be huge amounts of data if there were many generations, or if this was done across databases).  The way the API works in Atlas is: you get a list of generation results, and the user selects a single result to fetch.   

@gowthamrao what was the design intention of this ROhdsiWebAPI call?
